### ISSUE TITLE
Isssue with config.json

### ISSUE THREAD

**Issue by fa505305** [maddox/harmony-api#67]
Hello everybody,

My hub was not found and i am trying to force it with "hub_ip".
but this is still not working, could somebody check the config.json and help?

here it is: config.json
{
  "hub_ip": "10.10.10.11", "mqtt_host": "http://127.0.0.1",
  "mqtt_options": {
      "port": 1883,
      "username": "someuser",
      "password": "somepassword",
      "rejectUnauthorized": false,
      "hub_ip": "10.10.10.11"
  }
}

is this ok? if yes then i do not know where to search. the hub is reachable from the container 
thank you in advance
cheers

**Comment by maddox** [comment]
Try `mqtt://127.0.0.1` for the host.

**Comment by fa505305** [comment]
Hi,
nothing. no problem with host which seems to work fine. i can access Harmony API web page via local port 3775 (8282 container port). The problem is the HUB that is in different subnet than the host.
please find the logs.
FYI : running docker on synology, and i can ping the hub from the container.
please find the logs :
[harmony1csv.txt](https://github.com/maddox/harmony-api/files/1070413/harmony1csv.txt)

thank you in advance
fabien

**Comment by maddox** [comment]
Oh if you're running Harmony API in Docker, you have to use host networking for discovery to work. Manually setting the hub host doesn't work.

**Comment by fa505305** [comment]
host network is bridged so host IP and container IP are the same. access to different services via different ports.

**Comment by maddox** [comment]
Oh so even the host is on a different subnet from the harmony hub? Yeah that's going to be an issue. UPnP won't work across subnets.

I can try to get manual hub IP settings back in there, but I can't make any promises of when.

**Comment by fa505305** [comment]
yes correct and this is due to the nature of the hub that does not have an ethernet connection. I have different vlan for different services ... even at home and WIFI is separated ;-)

**Comment by fa505305** [comment]
It would be very nice if we could get manual hub IP.
thank you again for your replies

**Comment by ADN182** [comment]
I have the same problem, but in the same network. With Docker and i use host networking  (--net=host) but the Hub are not discover.

**Comment by ADN182** [comment]
Work for me with this docker : https://hub.docker.com/r/jawilson/harmony-api/ and config like that :   
{ 
"hubs": [
    {
      "name": "Living Room",
      "ip": "192.168.1.111"
    },
    {
      "name": "Bedroom",
      "ip": "192.168.1.112"
    },
  ]: 
}
